components/treeview/stories/treeview.stories.js

```diff
@@ -15,14 +15,22 @@ export default {
 	component: "TreeView",
 	argTypes: {
 		items: { table: { disable: true } },
-		variant: { table: { disable: true } },
+		variant: {
+			name: "Variant",
+			type: { name: "string" },
+			category: "Component",
+			defaultValue: { summary: "default" },
+			options: ["default", "detached"],
```

Comment:
Detached is the only non-default variant that I've included in the controls. Thumbnail is also considered a variant but that felt a little tricky since the thumbnails appear in the individual items. I'm leaving it out but I'm happy to hear any discussion or ideas related to this.

Reply:
Also, table does the thumbnails as a control. Would anything in the [table template file](https://github.com/adobe/spectrum-css/blob/rise-erpelding/css-871-fix-detached-variant/components/table/stories/template.js) be helpful? That control for table thumbnails is a boolean, but maybe it could be adapted for use here?


https://github.com/user-attachments/assets/7252529c-c2d2-4dc9-8a46-ff871e438819